src/library_html5_webgl.js

```diff
@@ -31,10 +31,10 @@ var LibraryHtml5WebGL = {
     }
 
     HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.alpha }}}>>2)] =
-    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.depth }}}>>2)] = 
-    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.antialias }}}>>2)] = 
-    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.premultipliedAlpha }}}>>2)] = 
-    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.majorVersion }}}>>2)] = 
+    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.depth }}}>>2)] =
+    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.antialias }}}>>2)] =
+    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.premultipliedAlpha }}}>>2)] =
+    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.majorVersion }}}>>2)] =
```

Comment:
Unrelated whitespace change?

(If this is the result of your editor doing this automatically, I would advise instead binding a key combination to remove trailing whitespace so you are in control over if/when it happens and your PR don't end up containing extra changes like this)

(No need to revert this now.. just a drive by comment)

Reply:
Yeah it is my editor doing it automatically. In most cases it's a useful setting to have by default (so it trims my own spaces w/o me having to remember to do so separately) and it's very rare for projects to have such trailing whitespace, but occasionally it does introduce such extra changes.

Maybe Emscripten could do a sweep do remove all of them at once?

Reply:
My advice would still be to not do this automatically... especially if you work on a lot of projects, but I guess you do and your experience has not been bad enough to change that setting :)

I used to have my ediitor setup that way but over time I realized is wanted more control, and I didn't want random extra changes being generated as part of my PRs.. but I guess it depends which projects you work on and how clean they are.

Reply:
> especially if you work on a lot of projects, but I guess you do and your experience has not been bad enough to change that setting :)

Yeah so far it's been almost always benefitial, with very occasional per-project settings to disable it (like 2 projects in the last couple of years maybe?).

Given that I'm used to working on Rust and JS where automatic formatting is pretty ubiquitous, trimming trailing whitespace has been the least risky thing I had, "auto format on save" can be more problematic 😅 I wish Emscripten enforced autoformatting too though as it takes a lot of unnecessary mental overhead for new contributors by automatically following the existing style of the project. I see it already has configs for both Clang-format and Python, but those are not really enforced.